docs/how-to-test-parcels-and-preview-scenes.md

```diff
@@ -1,27 +1,30 @@
 ## How to debug with local test parcels and preview scenes
 
-### Test Scenes
+### Running Test Scenes
 
-You can build the test scenes which are used by entering kernel with the `debug` mode:
+You can test and debug components and SDK functionalities with many test-scenes located at `kernel/public/test-scenes/`, running the client in `debug` mode.
+
+1. Test scenes must be built once locally before using them by running:
 
     make test-scenes
 
-If you want to run the client in `debug` mode, append the following query parameter to the URL:
+Note that the test-scenes building may take a while.
+
+2. To run the client in `debug` mode, append the following query parameter to the URL:
 
     http://localhost:3000/?DEBUG_MODE
 
-To spawn in a specific set of coordinates append the following query paramter:
+3. To spawn at the specific coordinates of a test scene append the position query paramter:
 
     http://localhost:3000/?DEBUG_MODE&position=10,10
 
 
+4. (optional) If you want the test scenes to be updated dynamically after the first build (to be able to modify the scenes code and see the changes by reloading the browser), you should run `make watch-only-test-scenes` in another terminal (as you'd also need a terminal running the regular `make watch` in parallel) and keep it watching for changes.
```

Comment:
just a suggestion/discussion starter (maybe?)
this option seems to be the simpler and probably the most used, also there is not much difference using this option instead of the previous one.
so... can we put it as the default option instead of optional?

Reply:
Good call, I think that's better too, take a look now at 
https://github.com/decentraland/explorer/blob/5eb95de5d6791ca2ea0e855609a7b0778c9164a4/docs/how-to-test-parcels-and-preview-scenes.md